main.py

```diff
@@ -499,11 +516,31 @@ def mapControlsMenu():
                 if key == pygame.K_ESCAPE and changingKey is False:
                     run = False
                     break
+                # checking if player asked for resetting controls
                 if key == pygame.K_r and changingKey is False:
                     for i in range(0, 14):
                         controls[i] = defaultControls[i]
                         text[i] = font.render(pygame.key.name(controls[i]), True, (255, 255, 255))
                     break
+                # checking which volume to switch
+                if key == pygame.K_i:
+                    if changingKey is False:
```

Comment:
remplace par un and ici

Reply:
ouais mais dupli de code c'est pas grave ?